sidecar/src/types.rs

```diff
@@ -75,7 +73,9 @@ impl Config {
     }
```

Comment:
Why the member name need to format to host:name?

Reply:
How come?

Reply:
sorry I read the code wrong